### DISCUSSION TITLE
Is it possible to set more than one table in a Backoffice page?

### DISCUSSION THREAD

**Discussion by calairoh** [mia-platform/community#301] [Q&A] [answered]
### Which section?

Backoffice

### Which is the version?

back-kit@1.3.17, micro-lc@2.2.0

### Description

I need to put more than one table in a Backoffice page. Each of them should have a dedicated `bk-crud-client` since every table ask for data coming from different collections.
I tried to put several `bk-table` objects, each of them with a specific `eventBus`, since they need to load data from different collections and so need to listen only to the `success` event of the specific `bk-crud-client`. However, this configuration does not work since all tables wait indefinitely due to the fact that `change-query` event is triggered on the "main" eventBus, thus not read by the tables and crud-client components. As a result, requests to backend are not sent and table keeps loading indefinitely.

After looking at the documentation, it seems that it's not possible to have more than one table in a page with `back-kit` standard web components. Is it correct?

### Actual Outcome

Table keeps waiting indefinitely since requests to backend are not triggered.

### Expected Outcome

Requests to backend are triggered and tables are populated.

**Comment by clabene** [comment, marked as answer]
Hi Davide,

the simplified configuration below exemplifies how multiple tables linked to different sources can be added.

The problem you are facing probably lies elsewhere. One possible point of interest is the property `reflectToUrl` of the `bk-crud-client`. Setting this prop to false disables `bk-crud-client` interaction with the page URL, which is recommended in case of multiple instances of the component are added to the configuration.

Could you please share the failing configuration?

```
{
  "definitions": {
    "dataSchema1": {
      "type": "object",
      "properties": {
        ...
      }
    },
    "dataSchema2": {
      "type": "object",
      "properties": {
        ...
      }
    }
  },
  "content": {
    ...
    {
      "properties": {
        "eventBus": "eventBus.pool.bus1",
        "dataSchema": {
          "$ref": "#/definitions/dataSchema1"
        }
      },
      "tag": "bk-table"
    },
    {
      "properties": {
        "eventBus": "eventBus.pool.bus2",
        "dataSchema": {
          "$ref": "#/definitions/dataSchema2"
        }
      },
      "tag": "bk-table"
    },
    {
      "properties": {
        "eventBus": "eventBus.pool.bus1",
        "basePath": "/path-1",
        "dataSchema": {
          "$ref": "#/definitions/dataSchema1"
        }
      },
      "tag": "bk-crud-client",
      "attributes": {}
    },
    {
      "properties": {
        "eventBus": "eventBus.pool.bus2",
        "basePath": "/path-2",
        "dataSchema": {
          "$ref": "#/definitions/dataSchema2"
        }
      },
      "tag": "bk-crud-client",
      "attributes": {}
    }
  }
}
```
